packages/components/src/inputs/src/InputGroup.tsx

```diff
@@ -0,0 +1,81 @@
+import { useResponsiveValue, useStyledSystem, type ResponsiveProp, type StyledComponentProps } from "@hopper-ui/styled-system";
+import { forwardRef, type ForwardedRef } from "react";
+import { useContextProps, Group as RACGroup, type GroupProps as RACGroupProps, composeRenderProps, InputContext, useSlottedContext } from "react-aria-components";
+
+import { SlotProvider, composeClassnameRenderProps, cssModule } from "../../utils/index.ts";
+
+import { InputGroupContext } from "./InputGroupContext.ts";
+
+import styles from "./InputGroup.module.css";
+
+export const GlobalInputGroupCssSelector = "hop-InputGroup";
+
+export interface InputGroupProps extends StyledComponentProps<RACGroupProps> {
+    size?: ResponsiveProp<"sm" | "md">;
+    isFluid?: ResponsiveProp<boolean>;
```

Comment:
as I mentioned above, we should probably add an isDisabled prop here and then add a data-disabled={isDisabled || undefined}

Reply:
i understand what you mean by simplifiying the selector, but the div around the input can not be "disabled". We are not owner of the input, the input is passed to us by a child, so we have to look at it's value, not try to own it